eng/tools/TypeSpecValidation/src/TypeSpecValidation.ts

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+
+import {exec} from "child_process";
+import fs from "fs";
```

Comment:
not a big deal if this is going to remain a simple file but it is better to use the `"fs/promises"` if you want to do check in parallel

Reply:
I will change this, I agree might as well get it right now